CMakeLists.txt

```
@@ -225,6 +225,10 @@ set(include_install_dir "include")
 
 set(generated_dir "${CMAKE_CURRENT_BINARY_DIR}/generated")
 
+if (NOT LIB_INSTALL_DIR)
+	set (LIB_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib")
+endif (NOT LIB_INSTALL_DIR)
+
```

I've tried your changes, and it sets `libdir` correctly, but `includedir` is still wrong:
```
$ head -3 evhtp.pc
prefix=/usr/local
libdir=/usr/local/lib
includedir=/evhtp
```
So, if we are to do this by restoring its previous behavior, then we should revert the next lines as well: 
```suggestion
if(NOT INCLUDE_INSTALL_DIR)
    set(INCLUDE_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/include)
endif()

```
This will get you a working `evhtp.pc` file:
```
 $ head -3 evhtp.pc
prefix=/usr/local
libdir=/usr/local/lib
includedir=/usr/local/include/evhtp
```